xtask/src/commands/changeset/matching_pull_request.rs

```diff
@@ -0,0 +1,99 @@
+#![allow(clippy::all, warnings)]
+pub struct MatchingPullRequest;
+pub mod matching_pull_request {
+    #![allow(dead_code)]
+    use std::result::Result;
+    pub const OPERATION_NAME: &str = "MatchingPullRequest";
+    pub const QUERY : & str = "fragment PrInfo on PullRequest {\n  url\n  number\n  author {\n    __typename\n    login\n  }\n  title\n  closingIssuesReferences(last: 4) {\n    nodes {\n      url\n      number\n      repository {\n        nameWithOwner\n      }\n    }\n  }\n  body\n}\nfragment PrSearchResult on SearchResultItemConnection {\n  issueCount\n  nodes {\n    __typename\n    ...PrInfo\n  }\n }\n\nquery MatchingPullRequest($search: String!) {\n  search(\n    type: ISSUE\n    query: $search\n    first: 1\n  ) {\n    ...PrSearchResult\n  }\n}\n" ;
```

Comment:
Isn't the graphql query you copy pasted in `xtask/src/commands/changeset/matching_pull_request.graphql` ? If it is then you should `include_str!`

Reply:
this is a generated file! (from that file)

i can add a comment to the query but can't add a comment to this file.

Reply:
(also i can't change this file to use include_str because it's generated.  (though that definitely seems nice in theory.)

Reply:
It looks to me like what you're trying to accomplish can be done with the derive macro, a gitignore and a build script (I'm assuming you would need a schema to generate the impl, but you don't want to check in the whole github graphql api schema)